package/MDAnalysis/lib/log.py

```diff
@@ -394,3 +396,34 @@ def echo(self, step, **kwargs):
             return
         echo(self.format_handler(format, vars(self)),
              replace=self.dynamic, newline=newline)
+
+
+# detect if running from jupyter, ipython or python
+try:
+    shell = get_ipython().__class__.__name__
+except: # non-interactive shell
+    from tqdm import tqdm as _tqdm_class
+else:
+    if shell == 'ZMQInteractiveShell': # Jupyter notebook and JupyterLab
+        from tqdm.notebook import tqdm_notebook as _tqdm_class
+    else: # Ipython
+        from tqdm import tqdm as _tqdm_class
+
+class ProgressBar(_tqdm_class):
+    r"""tqdm-like progress bar. Replaces the :class:`ProgressMeter` class to
```

Comment:
This is placeholder text and eventually needs to be replace by something more meaningful.

If you leave out the doc string, will it inherit the one from `_tqdm_class`?

Reply:
Yes, no doc-string = use the doc-string from the parent class. Should I do that instead ?

Reply:
yes